jujugui/static/gui/src/app/models/bundle.js

```diff
@@ -86,7 +86,7 @@ YUI.add('juju-bundle-models', function(Y) {
     */
     parseNameEmail: function(author) {
       var parts = /^([^<]+?) <(.+)>$/.exec(author);
-      if (Y.Lang.isNull(parts)) {
+      if (!parts) {
```

Comment:
Yeah, I originally had that as a null check (pretty sure exec returns null if there is no match) but I realised in this case it doesn't actually need to be a null check, just truthy.